projects/fluxcd/flux2/patches/0001-Update-go.mod-image-automation-controller-module-to-.patch

```diff
@@ -0,0 +1,97 @@
+From 8b9253e9379a2dafad4ba560162351a3a61e3b0c Mon Sep 17 00:00:00 2001
+From: Prow Bot <prow@amazonaws.com>
+Date: Mon, 30 Sep 2024 18:41:20 -0700
+Subject: [PATCH] Update go.mod image-automation-controller module to v0.38.0
+
```

Comment:
Can you add a brief description of the patch here? Keep it high-level and add a reference to the PR description